system/system_diagnostic_graph/example/example_0.yaml

```diff
@@ -9,24 +9,29 @@ nodes:
   - path: /autoware/modes/autonomous
     type: and
     list:
-      - { type: link, path: /functions/pose_estimation }
-      - { type: link, path: /functions/obstacle_detection }
+      - { type: link, link: /functions/pose_estimation }
+      - { type: link, link: /functions/obstacle_detection }
 
   - path: /autoware/modes/local
     type: and
     list:
-      - { type: link, path: /external/joystick_command }
+      - { type: link, link: /external/joystick_command }
 
   - path: /autoware/modes/remote
     type: and
     list:
-      - { type: link, path: /external/remote_command }
+      - { type: link, link: /external/remote_command }
 
   - path: /autoware/modes/emergency-stop
     type: debug-ok
```

Comment:
I think that debug can be read as a special meaning, so just ok or stale is fine.

Reply:
Fixed https://github.com/autowarefoundation/autoware.universe/pull/5447/commits/d0e47c983ac0a5cf7793001d0929dbde09ffced2